fcs/FSharp.Compiler.Service/FSharp.Compiler.Service.fsproj

```diff
@@ -46,6 +46,9 @@
     <EmbeddedText Include="$(FSharpSourcesRoot)\fsharp\FSComp.txt">
       <Link>FSComp.txt</Link>
     </EmbeddedText>
+    <EmbeddedText Include="$(FSharpSourcesRoot)\fsharp\Interactive.DependencyManager\InteractiveDependencyManager.txt">
+      <Link>FSComp.txt</Link>
```

Comment:
The link here overrides the link for FSComp.txt right above it

Reply:
Good, eyes, copy and paste error.

Thx